openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -43,6 +43,11 @@
       "type": "boolean",
       "default": true
     },
+    "processPiiSensitive": {
+      "description": "Optional configuration to turn off processing of column to apply pii senstive tags",
```

Comment:
Not sure I follow this. If it is True do we process?

Reply:
yes. It is by default set to true.

Reply:
Then maybe we can just say `"Optional configuration to automatically tag columns that might contain sensitive information"`

Reply:
I think that mixing a True/False setting with phrases such as on/off can be confusing